app/renderer/components/navigation/navigationBar.js

```diff
@@ -242,7 +246,7 @@ class NavigationBar extends ImmutableComponent {
         urlbar={this.props.navbar.get('urlbar')}
         onStop={this.onStop}
         menubarVisible={this.props.menubarVisible}
-        noBorderRadius={!isSourceAboutUrl(this.props.location)}
+        noBorderRadius={(isSourceAboutUrl(this.props.location) && this.isPublisherButtonEnabled) || this.isPublisherButtonEnabled}
```

Comment:
Yes we need it, because otherwise you get a rounded shape while loading some pages

![13](https://cloud.githubusercontent.com/assets/9574457/24895712/27bd1dfa-1e92-11e7-9948-b22b44862203.gif)